test/e2e_test.go

```diff
@@ -1000,7 +1011,7 @@ func TestAttachWithRekorBundle(t *testing.T) {
 
 func TestRekorBundle(t *testing.T) {
 	// turn on the tlog
-	defer setenv(t, env.VariableExperimental.String(), "1")()
+	t.Setenv(env.VariableExperimental.String(), "1")
```

Comment:
This and any other place in the tests that `VariableExperimental` is set can be removed, it no longer changes behavior (or if it did, that would be a bug)